### ISSUE TITLE
Image viewer cannot update while "generating predictions" it would seem

### ISSUE THREAD

**Issue by graeme-winter** [ccp4/DUI2#22]
At least I have had a `dials.index` step locked up for a few minutes with 

```
HTML Report + Reflection Prediction ... START
Generating HTML report
HTML Report ready
Generating Predictions
```

at the end of the log

![Screenshot 2022-09-07 at 16 19 17](https://user-images.githubusercontent.com/1241716/188916199-d9296970-0fff-4854-81ed-3b25d61eeec4.png)

The DUI2 process has also become extremely sluggish which I expect is related.

**Comment by graeme-winter** [comment]
Yes, `dials.predict` is occupying 100% of a CPU (which I don't need or expect right now)

**Comment by graeme-winter** [comment]
End of `stdout`

```
show_refl = True
None self.np_full_img
no mask to draw here

 url_dict = {'nod_lst': ['4'], 'cmd_lst': ['gmi None']} 

tmp_cmd2lst = ['gmi None']
replacing  gmi  with  get_mask_image

 cmd_lst:  [['get_mask_image', 'None']]

  err catch , wrong line, not sending IMG 

sending /*EOF*/
zlib. err catch (load_img_json_w_str)
show_refl = True
None self.np_full_img
no mask to draw here

 BrokenPipe err catch  while sending EOF or JSON 
```

**Comment by luisodls** [comment]
fixed the same way as #16

**Comment by graeme-winter** [comment]
That was not a fix
